DEVELOPING.md

```diff
@@ -16,7 +16,10 @@
 
 This extension is built using the [gradle-intellij-plugin](https://github.com/JetBrains/gradle-intellij-plugin/).
 
-1. Clone this repository: `git clone git@github.com:OpenLiberty/liberty-tools-intellij.git && cd liberty-tools-intellij`
+1. Clone this repository: `git clone git@github.com:OpenLiberty/liberty-tools-intellij.git`
+2. Clone the lsp4ij repository: `git clone git@github.com:MicroShed/lsp4ij.git`
+3. Build lsp4ij: `./gradlew jar`
+4. Save it in your local Maven repository: `mvn install:install-file -Dfile=build/libs/lsp4ij-0.0.5.jar -DgroupId=org.microshed -DartifactId=lsp4ij -Dversion=0.0.5 -Dpackaging=jar -DgeneratePom=true;`
```

Comment:
Should update these instructions to just `./gradlew publishToMavenLocal` as we're doing in the GitHub action.

Reply:
Ok